fvm/state/accounts.go

```diff
@@ -82,6 +82,26 @@ func (a *StatefulAccounts) AllocateStorageIndex(address flow.Address) (atree.Sto
 	copy(index[:], indexBytes[:8])
 	newIndexBytes := index.Next()
 
+	// store nil so that the setValue for new allocated slabs would be faster
+	// and won't do ledger getValue for every new slabs (currently happening to compute storage size changes)
+	// this way the getValue would load this value from deltas
+	// TODO: update this code to use atree.SlabIndexToLedgerKey method when the new version of atree is used
+	key := atree.LedgerBaseStorageSlabPrefix + string(index[:])
+
+	err = a.stateHolder.State().Set(string(address.Bytes()), "", key, []byte{}, false)
```

Comment:
is this the right way to convert to string here?

Reply:
noticed it's used like this everywhere else, so i guess so

Reply:
I think this one is fine, the cadence common.Address Bytes() method was problematic in the past